crates/rattler_conda_types/src/match_spec/parse.rs

```diff
@@ -249,13 +250,49 @@ fn parse_bracket_vec_into_components(
                 );
             }
             "fn" => match_spec.file_name = Some(value.to_string()),
+            "url" => {
+                // Is the spec an url, parse it as an url
+                let url = if parse_scheme(value).is_some() {
+                    Url::parse(value)?
+                }
+                // 2 Is the spec a path, parse it as an url
+                else if is_path(value) {
+                    let path = Utf8TypedPath::from(value);
+                    file_url::file_path_to_url(path)
+                        .map_err(|_error| ParseMatchSpecError::InvalidPackagePathOrUrl)?
```

Comment:
Do we need windows file handling absolute path shenanigans here again or is it using that?

Reply:
https://github.com/mamba-org/rattler/blob/aa710629c701eb7e10d5b42569beb5a8df445325/crates/file_url/src/lib.rs#L101

This should do all the magic.

Reply:
Aight!

Reply:
Yeah the `file_url::file_path_to_url` takes care of that.